programs/ziti-edge-tunnel/ziti-edge-tunnel.c

```diff
@@ -1996,6 +2160,772 @@ static void interrupt_handler(int sig) {
 }
 #endif
 
+#if __linux__
+static void diverter_update(char *ip, char *mask, char *lowport, char *highport, char *protocol, char *service_id, char *action){
+    bool state = true;
+    unsigned short random_port = 0;
+    unsigned char count = 0;
+    int rndm;
+    uv_random(NULL, NULL, &rndm, sizeof(rndm), 0, NULL);
+    random_port = htons(1024 + rndm % (65535 - 1023));
```

Comment:
Is `htons` (network byte order) really needed here? I could see it if the port value was going into a packet that gets put on the wire and received by hosts of any architecture, but I think we're just passing a command line option to a local process?

Reply:
No its not.  I had copied this function over from where I needed to pass it directly to an ebpf map key which stores the port in network order but here it can be in host order, so the htons can be omitted.